src/lib389/lib389/monitor.py

```diff
@@ -86,8 +87,52 @@ def get_statistics(self):
         starttime = self.get_attr_vals_utf8('starttime')
         return (dtablesize, readwaiters, entriessent, bytessent, currenttime, starttime)
 
-    def get_status(self, use_json=False):
-        return self.get_attrs_vals_utf8([
+    def get_resource_stats(self):
+        """
+        Get CPU and memory stats
+        """
+        stats = {}
+        pid = self._instance.get_pid()
+        total_mem = psutil.virtual_memory()[0]
+        p = psutil.Process(pid)
+        memory_stats = p.memory_info()
+
+        # Get memory & CPU stats
+        stats['total_mem'] = [str(total_mem)]
+        stats['rss'] = [str(memory_stats[0])]
+        stats['vms'] = [str(memory_stats[1])]
+        stats['mem_percent'] = [str(int(p.memory_percent("vms")))]  # Round up
+        stats['total_threads'] = [str(p.num_threads())]
+        stats['cpu_usage'] = [str(int(p.cpu_percent()))]  # Round up
```

Comment:
`round()` would provide more accurate percentages that `int()` (as `int()` will just discard a remainder)